### PR TITLE
completed css-four-column-layout exercise - Nick Sturz

### PR DESCRIPTION
**Pull request by nsturz**
Screen Shot 2022-07-05 at 11 07 20 PM

### CODE REVIEW

File: css-four-column-layout/styles.css [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+}
+
+.row {
+  display: flex;
+}
+
+.column-full,
+.column-one-fourth {
+  flex-basis: auto;
+}
```
Comment (codingcodymiller) [inline css-four-column-layout/styles.css:35]: This is not the proper way to make `column-` class rulesets. They should have a value that accurately represents the class name's stated size.

File: css-four-column-layout/styles.css [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+  justify-content: center;
+}
+
+.position-left {
+  left: 25%;
+}
+
+.relative {
+  position: relative;
+}
```
Comment (codingcodymiller) [inline css-four-column-layout/styles.css:55]: I don't see these in use anywhere, you should delete styling that is not being used.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: css-four-column-layout/styles.css (hunk 2 above)
Comment (nsturz) [reply css-four-column-layout/styles.css:55]: removed

File: css-four-column-layout/styles.css (hunk 1 above)
Comment (nsturz) [reply css-four-column-layout/styles.css:35]: was able to remove all CSS column rulesets altogether - styling can be achieved without them. Kept "column-one-fourth" classes  in HTML markup for reading purposes though.

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**